internal/tasks/set-env/set-env.yaml

```diff
@@ -0,0 +1,72 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: set-env
```

Comment:
no. nothing from this repo goes to the appsre internal clusters.

keep in mind, these pipelineruns are now going to be running in the same workspace at the release pipeline!

Reply:
> no. nothing from this repo goes to the appsre internal clusters.

That's not what @johnbieren told me yesterday. We have a plan to move all the internal pipelines to this repo to use the same promotion cadence and also enable tests for those tasks. And Johnny said this is already working.

I can see it here: https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/stonesoup/internal-services/cicd/stonesoup-internal-services.appsrep09ue1.yaml#L51

It uses the `internal/resources` because app interface just expects one directory where everything is stored. So the idea is that we will add a symlink for each task/pipeline yaml there.

So if you don't add the symlink, then yes, it won't get deployed. But the point is that everything under internal/ is meant to be deployed to the internal-services cluster.

Reply:
Yeah, what Martin said is correct. The point of the `internal` dir in this repo is to take the place of https://github.com/hacbs-release/app-interface-deployments/tree/main/internal-services/catalog because we have no automated promotion in the app-interface-deployments repo. If you don't want the resources deployed on the app-sre clusters, you can put it in the normal tasks/ dir instead of internal/tasks/

Reply:
I placed the pipeline and task in the correct directories now